pkg/core/text/main.go

```diff
@@ -34,14 +35,22 @@ func readFromURL(url string, line int) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	logrus.Debugf("HTTP GET to %q got a response with code %d", url, resp.StatusCode)
+	if resp.StatusCode > 399 {
+		logrus.Debugf("Status code for URL %q is not in the 1XX, 2XX or 3XX ranges: %d", url, resp.StatusCode)
```

Comment:
I think, I would make the message simpler but it's a minor suggestion

```suggestion
		logrus.Debugf("HTTP return code %q for URL %q", resp.StatusCode, url)
```